packages/connection-model/test/parse-and-build-uri.js

```diff
@@ -97,19 +97,38 @@ const tests = [
     connectionString:
       'mongodb://%40rlo:woof@localhost:27017/?' +
       'authMechanism=GSSAPI&readPreference=primary&' +
-      'authSource=%24external&ssl=false&authSource=$external',
+      'ssl=false&authSource=$external',
     expectedConnectionString:
       'mongodb://%40rlo@localhost:27017/?' +
       'authMechanism=GSSAPI&readPreference=primary&' +
-      'authSource=%24external&directConnection=true&ssl=false&authSource=$external'
+      'authSource=%24external&directConnection=true&ssl=false'
```

Comment:
Was about to ask why this was gone, then saw that it was here twice … wild :)

Reply:
And in the end I'm pretty sure we don't need to add it in there as the driver handles that automatically. But I didn't want to fiddle around with that, too.